src/fakeData/fakeHealthcareProfessionals.ts

```diff
@@ -44,15 +44,15 @@ function generateLocalizedNameInput(): gqlTypes.LocalizedNameInput {
     }
 }
 
-function generateDegreeInput(): gqlTypes.DegreeInput {
+export function generateDegreeInput(): gqlTypes.DegreeInput {
     return {
         nameEn: faker.person.jobTitle(),
-        nameJa: fakerJA.person.jobTitle(),
+        nameJa: '役職名',
```

Comment:
Also here fakerJa.person.jobTitle doesn't work for the Japanese faker, it generate a English job title

Reply:
this is fine for now 👍